core/src/test/scala/org/apache/spark/scheduler/BlacklistTrackerSuite.scala

```diff
@@ -88,6 +91,15 @@ class BlacklistTrackerSuite extends SparkFunSuite with BeforeAndAfterEach with M
     new TaskSetBlacklist(conf, stageId, clock)
   }
 
+  def configureBlacklistAndScheduler: Unit = {
```

Comment:
now that you're doing the equivalent thing in `beforeEach`, this entire method is unnecessary.  `beforeEach` gets called automatically for each test so you don't need to call this explicitly.

sorry I was unclear about this earlier.